### ISSUE TITLE
Are you willing to format the repository with `black` / `ruff` ?

### ISSUE THREAD

**Issue by baseplate-admin** [jmzambon/libreoffice-code-highlighter#30]
Hi,

The repository is awesome. But while working with it i noticed that the repository is not formatted with any code formatter.

I was wondering if you are willing to format it.

Thanks.

**Comment by flywire** [comment]
https://github.com/jmzambon/libreoffice-code-highlighter/commit/b071acbb2fa7aa3f292f015f69acd9ffb974e0ef#commitcomment-78692974

**Comment by baseplate-admin** [comment]
> https://github.com/jmzambon/libreoffice-code-highlighter/commit/b071acbb2fa7aa3f292f015f69acd9ffb974e0ef#commitcomment-78692974

makes sense to me. But instead of a workflow run it is one off thing in this case. But i am happy to close this

**Comment by jmzambon** [comment]
Hi,
I was on holidays.
Thank for the proposal, but strict code formatting seems dispensable here, as this is a very small project. I don't really want nor need such a new maintenance layer.
By the way, your pull request has modified all the Pygments files, which is not relevant as that code is developed and maintained elsewhere.

**Comment by baseplate-admin** [comment]
Hi, thanks for replying.

>  I was on holidays.

ah, no worries. I hope you had a good time.


---

>  By the way, your pull request has modified all the Pygments files, which is not relevant as that code is developed and maintained elsewhere.

makes sense to me. Let me open an issue in `pygments`. But i have a small question. How do you plan on keeping up with the latest edition of `pygments`?

> Thank for the proposal, but strict code formatting seems dispensable here, as this is a very small project. I don't really want nor need such a new maintenance layer.

It should be done automatically in the editor. You can configure your code editor to format your code on save.

---

**Comment by jmzambon** [comment]
Thank you for your advies.

> But i have a small question. How do you plan on keeping up with the latest edition of `pygments`?

For now, I just add an new release on each Pygments upgrade, replacing *pygments* folder as it is from the Pygments repository.

> It should be done automatically in the editor. You can configure your code editor to format your code on save.

I edit my code with Sublime Text. Probably a module exists for black/ruff. I'll have a look.

**Comment by baseplate-admin** [comment]
> For now, I just add an new release on each Pygments upgrade, replacing pygments folder as it is from the Pygments repository.

maybe [git submodules](https://www.git-scm.com/book/en/Git-Tools-Submodules) is a better way for this? that way you leverage dependabot to auto update the repository when there is a change in `pygments`

> I edit my code with Sublime Text. Probably a module exists for black/ruff. I'll have a look.

Maybe [python-black](https://packagecontrol.io/packages/python-black) is what you are looking for :D
